recipes/acado/all/conanfile.py

```diff
@@ -70,24 +71,43 @@ def build(self):
         cmake = self._configure_cmake()
         cmake.build()
 
+    @property
+    def _qpoases_sources(self):
+        return os.path.join("lib", "cmake", "qpoases")
+
     def package(self):
         self.copy("LICENSE.txt", src=self._source_subfolder, dst="licenses")
         cmake = self._configure_cmake()
         cmake.install()
+
         self.copy("*", src="lib", dst="lib")
-        tools.rmdir(os.path.join(self.package_folder, "share", "acado", "cmake"))
-        self.copy("qpoases.cmake", src="cmake", dst="cmake")
+        self.copy("qpoases.cmake", src="cmake", dst="lib/cmake")
+        qpoases_sources_from = os.path.join(self.package_folder, "share", "acado", "external_packages", "qpoases")
+        self.copy("*", src=qpoases_sources_from, dst=self._qpoases_sources)
+
+        tools.rmdir(os.path.join(self.package_folder, "share"))
 
     def package_info(self):
-        self.cpp_info.libs = ["acado_toolkit", "acado_casadi"]
+        if self.options.shared:
+            self.cpp_info.libs = ["acado_toolkit_s", "acado_casadi"]
+        else:
+            self.cpp_info.libs = ["acado_toolkit", "acado_casadi"]
 
         self.cpp_info.names["cmake_find_package"] = "ACADO"
         self.cpp_info.names["cmake_find_package_multi"] = "ACADO"
 
-        self.cpp_info.builddirs.append("cmake")
-        self.cpp_info.build_modules.append(os.path.join("cmake", "qpoases.cmake"))
+        self.cpp_info.builddirs.append(os.path.join("lib", "cmake"))
+        self.cpp_info.build_modules.append(os.path.join("lib", "cmake", "qpoases.cmake"))
 
         self.cpp_info.includedirs.append(os.path.join("include", "acado"))
-        self.cpp_info.includedirs.append(os.path.join("share", "acado", "external_packages", "qpoases"))
-        self.cpp_info.includedirs.append(os.path.join("share", "acado", "external_packages", "qpoases", "INCLUDE"))
-        self.cpp_info.includedirs.append(os.path.join("share", "acado", "external_packages", "qpoases", "SRC"))
+        self.cpp_info.includedirs.append(self._qpoases_sources)
+        self.cpp_info.includedirs.append(os.path.join(self._qpoases_sources, "INCLUDE"))
+        self.cpp_info.includedirs.append(os.path.join(self._qpoases_sources, "SRC"))
+
+    def validate(self):
+        if self.settings.compiler == "Visual Studio":
+            pass
+        elif self.settings.compiler == "gcc" and self.settings.compiler.libcxx != "libstdc++11":
+            raise ConanInvalidConfiguration("libstdc++11 required")
```

Comment:
this seems very suspicious 🤔

Reply:
Have a go at the error logs, I do not really know whats going on...

Reply:
please, provide more details about it. It seems you require C++11 standard, not libstdc++11